crates/nu-command/src/system/which_.rs

```diff
@@ -14,7 +14,6 @@ impl Command for Which {
         Signature::build("which")
             .input_output_types(vec![(Type::Nothing, Type::table())])
             .allow_variants_without_examples(true)
-            .required("application", SyntaxShape::String, "Application.")
```

Comment:
Can you update the documentation on `rest` to reflect something like `Application(s).`